src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -543,7 +543,7 @@ protected function getKernelParameters()
                 'kernel.debug' => $this->debug,
                 'kernel.name' => $this->name,
                 'kernel.cache_dir' => realpath($this->getCacheDir()) ?: $this->getCacheDir(),
-                'kernel.logs_dir' => realpath($this->getLogDir()) ?: $this->getLogDir(),
+                'kernel.logs_dir' => $this->getLogDir() ? (realpath($this->getLogDir()) ?: $this->getLogDir()) : false,
```

Comment:
should be `false !== $this->getLogDir() ? (realpath($this->getLogDir()) ?: $this->getLogDir()) : false`

Though I do not like the nested ternary operator. Building the value before and then only passing a variable here imo would increase readability a lot.